src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlFileStreamTest/SqlFileStreamTest.cs

```diff
@@ -256,44 +262,50 @@ private static void DropFileStreamDb(ref string fileStreamDir, string connString
             catch (SqlException e)
             {
                 Console.WriteLine("File Stream database could not be dropped. " + e.Message);
-                fileStreamDir = null;
             }
         }
 
-        private static string SetupTable(SqlConnection conn)
+        private static string SetupTable(string connString)
         {
             // Generate random table name
             string tempTable = "fs_" + Guid.NewGuid().ToString().Replace('-', '_');
```

Comment:
Could you use the utility function `DataTestUtility.GetUniqueNameForSqlServer` to generate a unique name?

Reply:
Done.